source/publishing_data/12_translating_dataset/translating_dataset.rst

```diff
@@ -0,0 +1,88 @@
+Translating a dataset
+=====================
+
+For domains with more than one language activated in the front office, Opendatasoft allows the translation of specific elements of the datasets, as well as their metadata. It should help users understand a dataset as best as possible, even though the data itself is not in their preferred language.
```

Comment:
I think it's a good place to be explicit about the effect it has, for example:
"Translated datasets will appear in the users' language when they're browsing your catalog; this will help users understand your dataset as best as possible, even though the data itself is not in their preferred language."